files-csharp/FastestWayToReadATextFileInCsharp/FastestWayToReadATextFileInCsharp/WaysToReadATextFileInCsharp.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Text;
+
+namespace FastestWayToReadATextFileInCsharp;
+
+public class WaysToReadATextFileInCsharp
+{
+    private readonly string _sampleFilePath;
+
+    public WaysToReadATextFileInCsharp(string sampleFilePath)
+    {
+        _sampleFilePath = sampleFilePath;
+    }
+
+    public string UseFileReadAllLines()
+    {
+        var lines = File.ReadAllLines(_sampleFilePath);
+        var stringBuilder = new StringBuilder();
+
+        foreach (var line in lines)
+        {
+            stringBuilder.AppendLine(line);
+        }
+        stringBuilder.Length -= Environment.NewLine.Length;
+
+        return stringBuilder.ToString();
+    }
+
+    public string UseFileReadAllText() => File.ReadAllText(_sampleFilePath);
+    
+    public string UseFileReadLines()
+    {
+        var lines = File.ReadLines(_sampleFilePath);
+        var stringBuilder = new StringBuilder();
+
+        foreach(var line in lines)
+        {
+            stringBuilder.AppendLine(line);
+        }
+        stringBuilder.Length -= Environment.NewLine.Length;
+
+        return stringBuilder.ToString();
```

Comment:
As this is repeated throughout the code many times, can we put it in a separate method and call it when appropriate?

Reply:
How do I show this in the article without breaking the flow of the article?